app/Http/Controllers/ArticlesController.php

```diff
@@ -0,0 +1,15 @@
+<?php
+namespace App\Http\Controllers;
+use App\Article;
+class ArticlesController extends Controller {
+    public function index()
+    {
+        $articles = Article::paginate(5);
+        return view('article.index', compact('articles'));
+    }
+  public function show($articleId)
+  {
+    $article = Article::find($articleId);
+    return view('article.show', compact('article'));
+  }
+}
```

Comment:
Format lại toàn bộ class này nhé. Trong WritingTask bọn mình viết theo chuẩn của Laravel 5 bạn ạ!
Còn indent của ``function show()`` đang sai bạn ạ!